docs/integration-guides/block-confirmation-tracking.md

```diff
@@ -0,0 +1,34 @@
+!!! tip "Guide based on node V19.0"
+	The recommendations below are based on node V19.0 and node versions earlier may not have all these options available. All integrations should upgrade their nodes to make use of easier block confirmation procedures detailed here.
+
+A primary function of any integration is to track confirmation of blocks on the network and the node provides both proactive notifications and options to request confirmation status on individual blocks. This combination allows building of robust systems for monitoring the status of any blocks of interest. 
+
+!!! tip "Notifications and fallback requests both recommended"
+	Due to notification methods not guaranteeing delivery of every block confirmed, it is recommended that manual requests for confirmation status be implemented as a fallback option. Both these types of methods are outlined below.
+
+### Receiving notifications of confirmation
+
+The recommended method for receiving notifications is via WebSockets through the confirmation topic. This method involves sending a subscribe command to start receiving notifications every time a block is confirmed by the network. The Type filtering options on this topic allow subscribing to only a subset of confirmations, but due to uncertainty about how confirmation of a block will occur (through direct voting or as a dependent for example) it is recommended that the default subscription be used to accept notifications for all confirmation types.
```

Comment:
Maybe we can italicize confirmation [topic], type [filtering]. I would also shorten the last sentence to keep it simple:

> It is recommended that the _type_ filtering options are not using for this purpose, to make it less likely to miss a notification.

Reply:
Just noticed my typo 🤦‍♂